### ISSUE TITLE
Break-down explanation expected in Interface Implementations for Enumerations

### ISSUE THREAD

**Issue by KySpace** [stefan-hoeck/idris2-elab-util#40]
In `Enum2.md` _Interface Implementation, Part 2_, stated "We will break this down in a moment. First, we check whether it actually works:"

And we never did ...

**Comment by stefan-hoeck** [comment]
You are right, of course. I will look into this and try to indeed break things up.
